src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -520,6 +520,12 @@ private static FunctionMetadata ParseFunctionMetadata(string functionName, JObje
                 functionMetadata.IsExcluded = (bool)value;
             }
 
+            if (configMetadata.TryGetValue("autoPackageRestoreEnabled", StringComparison.OrdinalIgnoreCase, out value) &&
+                value.Type == JTokenType.Boolean)
+            {
+                functionMetadata.IsAutoPackageRestoreEnabled = (bool)value;
```

Comment:
Why is this opt-in? Won't users just expect it to work w/o having to configure it? Even if we keep it opt-in, wondering why the setting would be per function as opposed to host level?